python.rst

```diff
@@ -822,6 +822,30 @@ Sources
   generated code and templates), but do not use globs as a shortcut to
   include files as sources.
 
+FLAG Guidelines
+===============
+
+Flag rules are related to core.flag library.
+
+Flag Naming
+-----------
+
+- Flag names should be lowercase (except in the case of secret values).
```

Comment:
This can and will change some time Real Soon Now. Maybe change the caveat (except when forced, such as with secrets).